platformio.ini

```diff
@@ -17,9 +17,11 @@ board_build.partitions = partitions_two_ota.csv
 board_build.embed_txtfiles =
   src/certs/aws_s3_root_ca_cert.pem
   src/certs/heroku_root_ca_cert.pem
-
+  components/esp32-wifi-manager/src/index.html
+  components/esp32-wifi-manager/src/style.css
+  components/esp32-wifi-manager/src/code.js
 build_flags =
-  '-DAPP_VERSION="0.5.0"'
+  '-DAPP_VERSION="0.6.0"'
   ; '-DWIFI_SSID="HUAWEI nova 3i"'
   ; '-DWIFI_PASS="w00tw00t"'
   '-DWIFI_SSID="TOPNET_D770"'
```

Comment:
Don't forget to remove password later and find a way to integrate it as GitHub secret in the ci/cd pipeline